cmd/paratime/add.go

```diff
@@ -5,75 +5,115 @@ import (
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
 
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
 
+	"github.com/oasisprotocol/cli/cmd/common"
 	cliConfig "github.com/oasisprotocol/cli/config"
 )
 
-var addCmd = &cobra.Command{
-	Use:   "add <network> <name> <id>",
-	Short: "Add a new ParaTime",
-	Args:  cobra.ExactArgs(3),
-	Run: func(cmd *cobra.Command, args []string) {
-		cfg := cliConfig.Global()
-		network, name, id := args[0], args[1], args[2]
-
-		net, exists := cfg.Networks.All[network]
-		if !exists {
-			cobra.CheckErr(fmt.Errorf("network '%s' does not exist", network))
-		}
-
-		pt := config.ParaTime{
-			ID: id,
-		}
-		// Validate initial paratime configuration early.
-		cobra.CheckErr(config.ValidateIdentifier(name))
-		cobra.CheckErr(pt.Validate())
-
-		// Ask user for some additional parameters.
-		questions := []*survey.Question{
-			{
-				Name:   "description",
-				Prompt: &survey.Input{Message: "Description:"},
-			},
-			{
-				Name: "symbol",
-				Prompt: &survey.Input{
-					Message: "Denomination symbol:",
-					Default: net.Denomination.Symbol,
-				},
-			},
-			{
-				Name: "decimals",
-				Prompt: &survey.Input{
-					Message: "Denomination decimal places:",
-					Default: fmt.Sprintf("%d", net.Denomination.Decimals),
+var (
+	symbol      string
+	numDecimals uint
+	description string
+
+	addCmd = &cobra.Command{
+		Use:   "add <network> <name> <id>",
+		Short: "Add a new ParaTime",
+		Args:  cobra.ExactArgs(3),
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := cliConfig.Global()
+			network, name, id := args[0], args[1], args[2]
+
+			net, exists := cfg.Networks.All[network]
+			if !exists {
+				cobra.CheckErr(fmt.Errorf("network '%s' does not exist", network))
+			}
+
+			pt := config.ParaTime{
+				ID: id,
+			}
+			// Validate initial paratime configuration early.
+			cobra.CheckErr(config.ValidateIdentifier(name))
+			cobra.CheckErr(pt.Validate())
+
+			paratimeInfo := struct {
+				Description string
+				Symbol      string
+				Decimals    uint8
+			}{}
```

Comment:
Actually you should populate it with `net.Denomination.Symbol,` and `net.Denomination.Decimals`.